crates/cxx-qt-gen/src/generator/naming/mod.rs

```diff
@@ -2,20 +2,10 @@
 // SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
 //
 // SPDX-License-Identifier: MIT OR Apache-2.0
-pub mod functions;
 pub mod method;
 pub mod namespace;
 pub mod property;
 pub mod qobject;
 pub mod signals;
 
 use syn::Ident;
```

Comment:
this seems unused now?

Reply:
I thought I had removed that and the functions file in the first commit?

Reply:
pretty sure this is already sorted